pkg/controller/controller.go

```diff
@@ -522,3 +522,27 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
+func isOwnedByBinding(objectMeta *metav1.ObjectMeta, binding *v1alpha1.Binding) bool {
+	ref := getControllerRef(objectMeta)
+	if ref == nil {
+		return false
+	}
+	return ref.APIVersion == binding.APIVersion &&
+		ref.Kind == binding.Kind &&
+		ref.Name == binding.Name &&
```

Comment:
Typed objects returned by typed clients do not have Kind and APIVersion set. So they should probably be hardcoded here.